### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by Bhavya2995**

### CODE REVIEW

File: src/actions/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { SIGNUP_REQUEST, SIGNUP_SUCCESS } from "./types";
+
+export const signUp = userDetails => async dispatch => {
+  dispatch({ type: SIGNUP_REQUEST, payload: userDetails });
+
+  const res = await fetch(
+    `https://secure-refuge-14993.herokuapp.com/add_user?username=${
```
Comment (arun-kumar-codes) [inline src/actions/index.js]: use base url from config file. ( create a config file)

File: src/actions/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { SIGNUP_REQUEST, SIGNUP_SUCCESS } from "./types";
+
+export const signUp = userDetails => async dispatch => {
+  dispatch({ type: SIGNUP_REQUEST, payload: userDetails });
+
+  const res = await fetch(
```
Comment (arun-kumar-codes) [inline src/actions/index.js]: create a service file which can be used to call api's

Comment (arun-kumar-codes) [reply src/actions/index.js]: use promise's instead of await

Comment (Bhavya2995) [reply src/actions/index.js]: Okay sir

File: src/reducers/userReducer.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { SIGNUP_REQUEST, SIGNUP_SUCCESS, SIGNUP_INFO } from "../actions/types";
+
+export default function(state = {}, action) {
```
Comment (arun-kumar-codes) [inline src/reducers/userReducer.js]: define initial state

**Pull request merged**